extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/HibernateValidatorBuildTimeConfig.java

```diff
@@ -0,0 +1,75 @@
+package io.quarkus.hibernate.validator.runtime;
+
+import io.quarkus.runtime.annotations.ConfigDocSection;
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
+public class HibernateValidatorBuildTimeConfig {
+
+    /**
+     * Enable the fail fast mode. When fail fast is enabled the validation
+     * will stop on the first constraint violation detected.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean failFast;
+
+    /**
+     * Method validation.
+     */
+    @ConfigDocSection
```

Comment:
I don't know what this does, but I absolutely trust that you do

Reply:
It adds a nice section in the doc table :).

Reply:
It creates a section like this one https://quarkus.io/guides/datasource#quarkus-datasource_quarkus.datasource.named-data-sources whose code is https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourcesJdbcRuntimeConfig.java#L22

Reply:
Of course @gsmet beats me to it ;-)

Reply:
More than I wanted to know 🙃